vendor/modules.txt

```diff
@@ -1101,8 +1101,8 @@ github.com/tonistiigi/dchapes-mode
 github.com/tonistiigi/fsutil
 github.com/tonistiigi/fsutil/copy
 github.com/tonistiigi/fsutil/types
-# github.com/tonistiigi/go-actions-cache v0.0.0-20240327122527-58651d5e11d6
-## explicit; go 1.20
+# github.com/tonistiigi/go-actions-cache v0.0.0-20241108014124-394979b8119e
+## explicit; go 1.22
```

Comment:
It does bump the minimum go version to go1.22, so if we backport, we should probably update the `vendor.mod` (and generated `go.mod` to go1.22 as well to be correct (FYI)